docs/Rakefile

```diff
@@ -80,6 +80,7 @@ task :serve, [:server_args] => [:init, :markdown_to_asciidoc, :adoc_variables] d
   if args[:server_args].nil? then
     sh "bundle exec middleman server"
   else
+    sh "mv -f source/using/images/minishift-architecture-template.png source/using/images/minishift-architecture.png"
```

Comment:
@budhrg If I am not wrong then if the image file not generated by any reason then even this `mv -f` also going to fail right? how we are sure that this is something going to work every time just changing the name of the file from `*.png.cache` to `*-template.png.cache`?

Reply:
@praveenkumar 

> If I am not wrong then if the image file not generated by any reason then even this `mv -f` also going to fail right? 

Yes, if generation is failed than `mv -f` is going to fail.

> how we are sure that this is something going to work every time just changing the name of the file from *.png.cache to *-template.png.cache?

I didn't that about this. Here we are renaming the template image name to actual image name which will be used in `basic-usage.adoc` file.

Reply:
@budhrg Can you please point me to the code which generates `minishift-architecture-template.png`